app/src/main/java/com/willowtree/vocable/presets/PresetsFragment.kt

```diff
@@ -29,6 +31,8 @@ class PresetsFragment : BaseFragment<FragmentPresetsBinding>() {
 
     private var maxCategories = 1
     private var maxPhrases = 1
+    private var isPortraitMode = true
+    private var isTabletMode = false
```

Comment:
Let me know if anyone objects to creating local variables in the PresetsFragment here for determining portrait mode / tablet mode...if in non-tablet mode and portrait mode, the phrases will automatically update with category forward and backward button.